### DISCUSSION TITLE
跪问一下大佬们

### DISCUSSION THREAD

**Discussion by ICEY183** [Melledy/Grasscutter#1195] [Q&A]
这个解压后怎么运行呀

**Comment by zhaodice** [comment]
https://github.com/Grasscutters/Grasscutter/wiki/Running#starting-the-server

**Reply by ICEY183** [reply]
> https://github.com/Grasscutters/Grasscutter/wiki/Running#starting-the-server

安装了java与dynamoDB环境，但执行script_cleanport.bat、script_serverboot.bat是没有弹窗反应，改了代理服务器后运行YuanShen.exe还是要输入mihoyo账号。怎么解决呢Σ(ﾟдﾟ;)

**Reply by ICEY183** [reply]
![微信截图_20220605202430](https://user-images.githubusercontent.com/106876342/172050292-89055c06-5675-4438-ad90-b62bbfe57c74.png)
可能是MongoDB连接问题?点了connect后显示No saved queries yet.

**Comment by zhaodice** [comment]
这还真不清楚，我是在linux上搭建的

**Comment by ChisatoNishikigi73** [comment]
其实可以不用创建数据库，开启服务后运行核心，它会自动创建并连接数据库
